src/link/style.scss

```diff
@@ -301,6 +301,19 @@ body {
 				animation-name: wikipediapreview-edit-inline-loader-expanded;
 			}
 		}
+
+		&-tooltip {
+			width: 190px;
+			height: 40px;
+			background: #36c;
+			color: #fff;
+			display: flex;
+			justify-content: center;
+
+			&-text {
+				align-self: center;
+			}
+		}
```

Comment:
<img width="211" alt="image" src="https://github.com/wikimedia/wikipediapreview-wordpress/assets/2560096/00e1a3ee-34fc-4c8b-ae4f-d5bbfeadcdb7">

is that possible to color the arrow as well?

Reply:
Turns out it is possible after finding the right target classes: https://github.com/wikimedia/wikipediapreview-wordpress/pull/111/commits/03f74a54b535b65a5c1897ac8907e89c86b59d8e